app/build.gradle

```diff
@@ -0,0 +1,45 @@
+plugins {
+    id 'com.android.application'
+    id 'kotlin-android'
+}
+
+android {
+    compileSdkVersion 30
+    buildToolsVersion "29.0.3"
```

Comment:
It's better to delete the line as it doesn't match the SDK version and Android Gradle Plugin has default buildToolsVersion included.

Reply:
It is a project sample, does not include any best practices here.